react/features/chat/components/native/ChatButton.js

```diff
@@ -36,7 +43,9 @@ class ChatButton extends AbstractButton<Props, *> {
      * @returns {void}
      */
     _handleClick() {
-        this.props.dispatch(openChat());
+        const { dispatch } = this.props;
+
+        return ConferenceNavigationContainerRef.navigate('Chat') && dispatch(openChat());
```

Comment:
Why do we need openChat ? The navigation should make the screen visible, right?

Reply:
Removed it. I thought isOpen from state would be used in other parts of the app, but no.